string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,13 @@
+/* exported isVowel */
+function isVowel(char) {
+  switch (char.toLowerCase()) {
+    case 'a'.toLowerCase():
+    case 'e'.toLowerCase():
+    case 'i'.toLowerCase():
+    case 'o'.toLowerCase():
+    case 'u'.toLowerCase():
```

Comment:
There's no need to include `.toLowerCase()` here, because you already did it in your conditional on line 3